.env.example

```diff
@@ -0,0 +1,12 @@
+POSTGRES_HOST = postgres.example.com
+POSTGRES_PORT = 5432
+POSTGRES_USER = postgres.user
+POSTGRES_PASSWORD = postgres.password
+POSTGRES_DATABASE = postgres.database
+POSTGRES_SSL = true
+
+CATALOG_API = catalog.example.com
```

Comment:
CATALOG_API should be CATALOG_API_HOST for consistency with the other host variable names

Reply:
done

Reply:
😄